src/runner/setup.rs

```diff
@@ -7,23 +7,33 @@ use crate::prelude::*;
 
 const VALGRIND_CODSPEED_VERSION: &str = "3.21.0-0codspeed1";
 
+/// Run a command with sudo if available
+fn run_with_sudo(command_args: Vec<&str>) -> Result<()> {
```

Comment:
Probably, this should return the built command without running it, giving the flexibility to anyone using the function to chose how the execution should go (ex: maybe I want to throw away stdout?)